packages/itwin/tree-widget/src/components/tree-header/TreeHeader.scss

```diff
@@ -82,7 +76,7 @@
       width: calc(100% - var(--enlarged-header-height));
 
       &.contracted {
-        width: var(--enlarged-header-height);
+        width: unset;
```

Comment:
am I right to think that our e2e tests don't capture the effect of this? shouldn't they?

Reply:
Borderless button with matches `var(--enlarged-header-height)` so there are no difference. Changed to `unset` to match styling of default density.